app/views/notes/_title_suggestion.html.erb

```diff
@@ -1,6 +1,7 @@
 <div class="alert alert-warning">
-	<a href="/profile/<%= user %>"><%= user %></a> is suggesting an alternative title, "<%= title %>".
+        <a href="/profile/<%= user %>"><%= user %></a> <%= translation('notes._title_suggestion.suggesting_title') %> "<%= title %>".
```

Comment:
Hi @Angelvicks, this is the only file that needs to be changed. The rest are not needed.

Also, could you remove the empty line on line 3 and indent line 2 to match the other lines? Thanks!